components/Preview/PostsPreview.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Card } from '@sanity/ui'
+
+export function PostsPreview(props: any) {
+  return (
+    <Card scheme="light" style={{ width: '100%', height: '100%' }}>
+      <iframe style={{ width: '100%', height: '100%' }} src={getUrl(props)} />
+    </Card>
+  )
+}
+
+function getUrl({ document }) {
```

Comment:
I was thinking about what you mentioned @stipsan about moving this so it can be both used here and on the sanity.config.ts but I've simplified it for the needs in the PostPreview 🤔 it's like you said, we can keep it for now as we work and do a larger pass later 🤔

Reply:
Sounds good 👍